internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -44,7 +44,7 @@ type KopsProvisioner struct {
 }
 
 // NewKopsProvisioner creates a new KopsProvisioner.
-// TODO(gsagula): Consider replacing all these paramaters with a struct for readability.
+// TODO(gsagula): Consider replacing all these parameters with a struct for readability.
```

Comment:
since you are changing this file you can remove the username from the todo 
```suggestion
// TODO: Consider replacing all these parameters with a struct for readability.
```